server/src/callbacks.mjs

```diff
@@ -68,3 +57,28 @@ Empirica.onRoundEnd(function ({ round }) {});
 Empirica.onGameEnd(function ({ game }) {
   
 });
+
+
+
+Empirica.onNewBatch(async function ({ batch }) {
+  batch.set("topics", {});
```

Comment:
It might make sense to move this down below the `treatments.forEach` to keep it together with the code that populates it?